modules/ROOT/pages/bat-bdd-reference.adoc

```diff
@@ -86,9 +86,12 @@ A test suite must contain at least one test (.dwl) file.
 
 == Keeping Confidential Information Secure in Tests
 
-In your tests, you can access shared secrets in Anypoint Secrets Manager, which enables you to securely store sensitive information, such as a password, authentication token, endpoint URL, or webhook URL.
+In your tests, you can access shared secrets that are symmetric key types from Anypoint Secrets Manager, which enables you to securely store sensitive information, such as a password, authentication token, endpoint URL, or webhook URL
```

Comment:
```suggestion
In your tests, you can access shared secrets that are symmetric key types from Anypoint Secrets Manager. Secrets Manager enables you to securely store sensitive information, such as a password, authentication token, endpoint URL, or webhook URL.
```

Reply:
Hey, it might be good to break up that first sentence. Otherwise, this looks good.